jdaviz/core/template_mixin.py

```diff
@@ -1413,11 +1417,9 @@ def _on_subset_created(self, msg=None):
         viewer = self.viewer if isinstance(self.viewer, list) else [self.viewer]
         for current_viewer in viewer:
             for layer in self._get_viewer(current_viewer).state.layers:
-                if layer.layer.label == new_subset_label:
-                    # Is it ok if only one subset layer has this callback?
+                if layer.layer.label == new_subset_label and is_not_wcs_only(layer.layer):
                     layer.add_callback('color', self._update_layer_items)
                     layer.add_callback('visible', self._update_layer_items)
-                    break
```

Comment:
this had to be removed in order for mixed state of new subsets to be shown correctly on the tabs